presto-main/src/main/java/com/facebook/presto/operator/LookupJoinOperator.java

```diff
@@ -342,6 +342,7 @@ private void tryUnspillNext()
             // Close previous lookupSourceProvider (either supplied initially or for the previous partition)
             lookupSourceProvider.close();
             lookupSourceProvider = new StaticLookupSourceProvider(lookupSource);
+            statisticsCounter.updateLookupSourcePositions(lookupSource.getJoinPositionCount());
```

Comment:
@dain , as @losipiuk pointed out, this means we can count partition that was spilled _during_ joining twice, overestimating lookup source positions in the operator stats. We can't just count spilled and not spilled partitions once, since LJO can start work before a spilled partition's input is fully consumed. The overestimation can be mitigated by:
* moving the info to HBO operators stats _or_
* counting in PLSF and updating the number in LJO's `statisticsCounter` at the end (or multiple times, if this number is important _during_ execution)

Before doing anything with this, i'd like to hear your opinion and whether the overestimation is real problem.

Reply:
I was going to ask "is overestimation is a real problem" also :)  If not, add a comment to this code and the getter method for "LookupSourcePositions" that the number is an estimate.

Reply:
I added relevant comments. I couldn't find usages